app/presenters/gui_presenter.rb

```diff
@@ -0,0 +1,29 @@
+class GuiPresenter
+
+  def read_gui_yaml
+    puts "READING YML"
```

Comment:
This looks like debugging cruft; can we drop it?

Reply:
oops, debug stuff yeah :(